coops-ui/src/components/EntityInfo.vue

```diff
@@ -58,6 +69,7 @@
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
 import { mapState, mapGetters } from 'vuex'
+import { EntityStatus } from '@/enums'
```

Comment:
Should we add comments here for import consistency? ie // Enums , // Libraries ?  I started doing that before, not sure if was something you prefer though, thoughts on that?

Reply:
Yes, I liked that you did that in other components that imported many things, but this component only has 3 imports and I thought the code was self-documenting :)

Reply:
Makes sense, it certainly is!